packages/recomponents/src/components/r-input/r-input.vue

```diff
@@ -65,18 +70,20 @@
                     @focus="focus"
                     @click="click"
                     :name="name"
+                    :maxlength="maxLength"
                     :autocomplete="autocompleteFlag"/>
-                <r-icon :icon="rightIcon" v-if="rightIcon"></r-icon>
+                <r-icon :icon="rightIcon" v-if="rightIcon" :class="{'cursor-pointer': rightIconClickPointer}" @click.stop="$emit('right-icon-click')"></r-icon>
             </div>
             <slot name="right-button"/>
-            <div class="r-field-addon no-flex" v-if="rightLabel">{{rightLabel}}</div>
+            <div class="r-field-addon no-flex text-muted" v-if="rightLabel">{{rightLabel}}</div>
         </div>
         <span class="r-field-caption" v-if="helpText || maxLength">{{helpText}} <span v-if="maxLength">{{charactersLeft}}</span></span>
     </div>
 </template>
 
 <script>
     import shortid from 'shortid';
+    import '../../directives/r-fs-block';
```

Comment:
I think it's better to import directives globally rather than per component.

Reply:
sure, https://github.com/Rebilly/rebilly-recomponents/pull/54/commits/da0e668b833d8e99914312f6aff81e4104f0bd72